### PR TITLE
Shutdown refactor

### PR DESCRIPTION
**Pull request by magnetto90**

### CODE REVIEW

**Review by EperezOk — DISMISSED** [review]

**Review by TomasCImach — DISMISSED** [review]
- remove `notShutdown` modifier from `shutdown()` function:
  - when isShutdown == false -> the function is callable. No changes here.
  - when isShutdown == true -> shutdown() function will not modify the state of the contract.
  
- removing `uint256 fraction` feeding pos.cost() with ONE directly. No point to cache.

**Review by EperezOk — DISMISSED** [review]
I would change the name of the `ORACLE` constant to `FEED` for consistency with the rest of the codebase.

**Comment by magnetto90** [comment]
> I would change the name of the `ORACLE` constant to `FEED` for consistency with the rest of the codebase.

Done, good idea!

File: tests/OverlayV1Market.t.sol [hunk 1]
```diff
@@ -107,25 +108,66 @@ contract MarketTest is Test {
...
+        vm.startPrank(USER);
+        vm.expectRevert("OVLV1: shutdown");
+        market.build(1e18, 1e18, true, type(uint256).max);
+        vm.expectRevert("OVLV1: shutdown");
+        market.unwind(1, 1e18, 0);
+        vm.expectRevert("OVLV1: shutdown");
+        market.liquidate(USER, 1);
+        
+        market.emergencyWithdraw(1);
+        market.emergencyWithdraw(2);
```
Comment (TomasCImach) [inline tests/OverlayV1Market.t.sol:170]: add assert of the emergencyWithdraw amount to be equal to the intialCollateral * remainingFraction

Comment (TomasCImach) [reply tests/OverlayV1Market.t.sol:170]: and assert that after all the unwinds, the OV balance of the Market is 0

Comment (magnetto90) [reply tests/OverlayV1Market.t.sol:166]: Done!

**Review by EperezOk — APPROVED** [review]

**Pull request merged**